### PR TITLE
feat: DLT-3352 refresh GEO standard, publish llms.txt, add freshness check

### PR DESCRIPTION
**Pull request by belumontoya**
## Summary

Refreshes the GEO standard with 2026 practices, publishes `/llms.txt` and `/llms-full.txt` from `dialtone.dialpad.com` via `@vuepress/plugin-llms`, and adds a non-blocking `last_verified` + 90-day PR-comment freshness check.

**Jira:** [DLT-3352](https://dialpad.atlassian.net/browse/DLT-3352)
**Initiative:** [DLT-3109](https://dialpad.atlassian.net/browse/DLT-3109) (dialtone-ai-docs)

## Why

The standard was 7 weeks stale, didn't mention `llms.txt` (now mainstream — Anthropic, Cloudflare, Vercel, Coinbase, Cursor publish), and had no mechanism to surface drift. AI citations decay in ~13 weeks per 2026 research; 76.4% of ChatGPT's top-cited pages are <30 days old. We needed all three problems solved together because they share one purpose: a living standard that produces a concrete public-facing artifact and stays current automatically.

## What's in each commit

1. **`feat(dialtone-docs)`** — `last_verified` field spec + populate on all standards/, refreshed GEO standard content (llms.txt section, Content Freshness section, Platforms differ callout, refreshed schema example, 2026 Resources), `check-freshness.mjs` script with 11 Vitest unit tests
2. **`feat(dialtone-documentation)`** — `@vuepress/plugin-llms@2.0.0-rc.112` (matches existing plugin versions) with curated filter (foundations + section indexes + key guides) and branded `llmsTxtTemplateGetter`
3. **`feat(workflow, dialtone-docs)`** — `standards-freshness.yml` workflow that posts a PR comment listing stale docs (non-blocking; only triggers on PRs touching `standards/`); CLAUDE.md note under Doc Sync Tooling
4. **`chore(dialtone-documentation)` NO-JIRA** — Pre-existing markdownlint errors fixed in two files I touched during verification (`mcp-server/index.md`, `line-clamp.md`). Separable; reviewable on its own.

## Local verification

- `pnpm nx run dialtone-docs:test` — 41/41 pass (was 30; added 11 freshness tests)
- `pnpm nx run dialtone-docs:check-freshness` — 0 stale (all standards at `last_verified: 2026-04-27`)
- `pnpm nx run dialtone-documentation:build` — succeeds; generates `/llms.txt` (17 entries, 2.8 kB) and `/llms-full.txt` (78.9 kB) at `.vuepress/dist/`

## What CI verifies that local couldn't

- TS-002: PR comment from `standards-freshness.yml` when a standard has stale `last_verified`
- TS-003: Workflow does NOT trigger when standards/ are untouched (this PR includes `standards/` changes, so we'll see the workflow run; check the PR's Checks tab)

## Out of scope / known

- **Pre-existing markdownlint errors** in 4 files I never touched: `docs/about/whats-new/posts/2025-12-2.md`, `docs/components/button.md`, `docs/components/modal.md`, `docs/guides/content/action-language/index.md`. ~19 errors. Tech debt that existed on staging before this branch — needs a dedicated lint cleanup PR.
- **Wider `last_verified` adoption** to other content directories (`architecture/`, `development/`, `reference/`, `workflows/`) deferred per Batch 1 decision; revisit after standards-only proves out.
- **Scheduled freshness re-run** and **substance-guard tooling** deferred per PRD; revisit if drift recurs.

## Posture

Follows additive-over-replacement: existing Claude doc-sync hooks/skill stay byte-for-byte unchanged; this is a parallel, tool-agnostic surface for the GEO standard specifically.

[DLT-3352]: https://dialpad.atlassian.net/browse/DLT-3352?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DLT-3109]: https://dialpad.atlassian.net/browse/DLT-3109?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/dialtone-docs/src/generators/check-freshness.mjs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  const dateStr = parseLastVerified(lastVerified);
+  if (!dateStr) return true;
+  const verifiedDate = new Date(dateStr);
+  if (isNaN(verifiedDate.getTime())) return true;
+  const diffDays = Math.floor((today.getTime() - verifiedDate.getTime()) / (1000 * 60 * 60 * 24));
+  return diffDays > thresholdDays;
+}
+
+async function check() {
+  const today = new Date();
```
Comment (braddialpad) [inline packages/dialtone-docs/src/generators/check-freshness.mjs]: minor timezone issue here:

new Date('YYYY-MM-DD') parses as UTC midnight; new Date() is local wall clock. On a server with a negative UTC offset, a boundary date (exactly 90 days ago) could appear 91 days old if 
  the script runs before midnight UTC. For a 90-day threshold this is practically harmless, but the tests also hardcode new Date('2026-04-27') which parses as UTC midnight — meaning the  
  test environment doesn't surface the inconsistency. Fix by normalizing to UTC-only arithmetic:    
                                                                                       
```
  const today = new Date();
  const todayUTC = Date.UTC(today.getUTCFullYear(), today.getUTCMonth(), today.getUTCDate());                                                                                              
  // then compare against verifiedDate.getTime()
```

**Review by braddialpad — APPROVED** [review]

File: .github/workflows/standards-freshness.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+name: Standards Freshness Check
+
+on:
+  pull_request:
+    paths:
+      - 'packages/dialtone-docs/src/content/standards/**'
```
[the remark this answers is not in the document]
Comment (belumontoya) [reply .github/workflows/standards-freshness.yml:6]: Sticking with `pull_request` + `continue-on-error: true` on the comment step. Fork PRs skip the comment without failing the job, which gets us the same end-state without the privileged-context risks of `pull_request_target` (write-token in base context, easy supply-chain footgun if the workflow ever evolves to check out PR code). The freshness check is for internal contributors editing `standards/`, so fork PRs are an edge case here.

File: packages/dialtone-docs/project.json [hunk 3]
```diff
@@ -8,6 +8,10 @@
     "test": {
       "executor": "nx:run-script",
       "options": { "script": "test" }
+    },
+    "check-freshness": {
+      "executor": "nx:run-script",
+      "options": { "script": "check-freshness" }
     }
```
[the remark this answers is not in the document]
Comment (belumontoya) [reply packages/dialtone-docs/project.json:15]: The CI workflow invokes `node packages/dialtone-docs/src/generators/check-freshness.mjs` directly — not via Nx — so caching wouldn't apply on the workflow run regardless. The Nx target exists for ad-hoc local runs (`pnpm nx run dialtone-docs:check-freshness`), where caching a one-shot diagnostic isn't a real concern. Leaving as-is.

File: packages/dialtone-docs/src/generators/check-freshness.mjs [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+    checkedFiles.push(filePath);
+
+    if (isStale(rawVal, today, THRESHOLD_DAYS)) {
+      const parsed = lastVerifiedStr ? new Date(lastVerifiedStr) : null;
+      const daysSinceVerified = parsed
+        ? Math.floor((today.getTime() - parsed.getTime()) / (1000 * 60 * 60 * 24))
+        : null;
+      stale.push({ file: filePath, last_verified: lastVerifiedStr, days_since_verified: daysSinceVerified });
+    }
```
[the remark this answers is not in the document]
Comment (belumontoya) [reply packages/dialtone-docs/src/generators/check-freshness.mjs:76]: Acknowledged — both compute the same diff. Leaving as-is for this PR: the duplication is six lines apart with the same formula, and refactoring `isStale` to also return the day count adds API surface for one consumer. Easy to revisit if it drifts.

File: packages/dialtone-docs/src/generators/check-freshness.mjs (hunk 1 above)
Comment (belumontoya) [reply packages/dialtone-docs/src/generators/check-freshness.mjs]: Good catch — fixed in the next commit. Anchored `today` to UTC midnight at the entry of `check()` so both operands of the date diff are UTC dates, matching what `new Date('YYYY-MM-DD')` already produces for `last_verified`. Same shape as your suggestion.
